Task2_3_TrainingClass.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Task2_3
+{
```

Comment:
ILesson interface in not  good decision here )) I will explain on lecture the difference ))

Reply:
thanks